docs/skopeo-copy.1.md

```diff
@@ -66,6 +66,15 @@ MANIFEST TYPE (oci, v2s1, or v2s2) to use in the destination (default is manifes
 
 Print usage statement
 
+**--multi-arch**
+
+If _source-image_ refers to an index of images for multiple architectures, this option controls what is copied. Default is system.
+
+Options:
+- system: Copy the image that matches the system architecture instead of the index
+- all: Copy all of the images in the index and the index itself
+- index-only: Copy only the index itself
```

Comment:
Add something like “… . Note that this usually fails unless the referenced per-architecture images are already present in the destination.”?